intl/messages/en-US.json

```diff
@@ -1,5 +1,8 @@
 {
     "home.title": "Homepage",
+
     "contacts.name": "Name: { name }",
-    "contacts.email": "Email: { email }"
+    "contacts.email": "Email: { email }",
+
+    "splash.error": "Oops, something went bad. Please refresh to try again."
```

Comment:
I believe this message is related to https://github.com/moxystudio/next-with-moxy/pull/58 and not to this PR.

```suggestion
```